packages/lab/src/dialog-next/DialogNext.css

```
@@ -0,0 +1,63 @@
+/* Styles applied to DialogNext component */
+.saltDialogNext {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding-top: var(--salt-spacing-300);
+  padding-bottom: var(--salt-spacing-300);
+  background: var(--salt-container-primary-background);
+  max-height: 100%;
+  max-width: 100%;
+  box-shadow: var(--salt-overlayable-shadow-modal);
+  min-width: calc(10 * var(--salt-size-base));
+  max-width: calc(20 * var(--salt-size-base));
+  z-index: calc(var(--salt-zIndex-appHeader) - 1);
+  overflow-y: auto;
+}
+
+/* Styles applied to DialogNext background overlay */
+.saltDialogNext-overlay {
+  background: var(--salt-overlayable-background);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: var(--salt-spacing-400);
+  z-index: var(--salt-zIndex-dialogNext);
```

```suggestion
  z-index: var(--salt-zIndex-dialog;
```
i assume was a find and replace

---

Yep, good spot 🙌